src/java.base/share/classes/java/lang/classfile/TypeKind.java

```diff
@@ -97,10 +148,32 @@ public TypeKind asLoadable() {
         };
     }
 
-    TypeKind(String name, String descriptor, int newarrayCode) {
+    TypeKind(String name, String descriptor, int newarrayCode,
+             Opcode load0, Opcode load1, Opcode load2, Opcode load3, Opcode load, Opcode loadw,
+             Opcode store0, Opcode store1, Opcode store2, Opcode store3, Opcode store, Opcode storew,
+             Opcode returnOpcode, Opcode arrayLoadOpcode, Opcode arrayStoreOpcode
```

Comment:
Note that while these can increase the performance of lookups, we need to check the initialization overhead caused by obtaining all these Opcode.